packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -53,9 +53,10 @@ _ref:
             - `pages: object[]`: An array of page objects.
 
             Pages are made up of blocks. Blocks have the following basic schema:
-            - `id: string`: __Required__ - A unique identifier for a block.
-            - `type: string`: __Required__ - This is the block type identifier and defines what block to used.
-            - `properties: object`: All the settings passed to a block component.
+            - `id: string`: __Required__ - An identifier for a block.
```

Comment:
Block id is no longer required

Reply:
Change made in https://github.com/lowdefy/lowdefy/commit/386463869c78c494c05636e1e435419c96917b02